frontend/src/cljs/imas_seamap/map/subs.cljs

```diff
@@ -78,6 +82,20 @@
   (let [categories (get-in db [:map :categories])]
     (map-on-key categories :name)))
 
+(defn national-layer-year [db _]
+  (:year
+   (or ; we use 'or' here instead of having an additional param in 'get-in' because get-in will actually read nil as a success and return that
```

Comment:
Makes sense (common-lisp has two return values; the value itself, and if the key was found -- which gets around this issue).  Might be worth thinking about if the key should be present if it's nil, I guess.